### ISSUE TITLE
RX 570 unrecognised "card 0 Unsupported AMD GPU"

### ISSUE THREAD

**Issue by pcsimple** [azeam/powerupp#25]
- Hello my RX 570 is not recognised by powerupp but it is by upp.
- My card have a modified bios is-it the problem?
- Thanks

**Comment by azeam** [comment]
No, only 5000 and 6000 series support has been implemented in powerupp. The Polaris PowerPlay table structure is quite different from Navi and it would require some work, for which time is lacking at the moment. Happy to accept PRs though.

**Comment by sibradzic** [comment]
Right, powerupp does not support Polaris. However, it is supported in https://github.com/sibradzic/upp & https://github.com/sibradzic/amdgpu-clocks.
